engines/dfc_provider/app/serializers/dfc_provider/supplied_product_serializer.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+# Serializer used to render a DFC SuppliedProduct from an OFN Product
+# into JSON-LD format based on DFC ontology
+module DfcProvider
+  class SuppliedProductSerializer
+    def initialize(product)
+      @product = product
+    end
+
+    def serialized_data
+      {
+        "@id" => "/products/#{@product.id}",
+        "dfc:hasUnit" => {
+          "@id" => "/unit/#{@product.unit_description.presence || 'piece'}",
+          "rdfs:label" => "#{@product.unit_description.presence || 'piece'}"
+        },
+        "dfc:quantity" => @product.total_on_hand,
```

Comment:
this is legacy spree that we should delete.
the correct way is product.on_hand

Reply:
btw, I wonder if we should even have this in the system. The variants will most of the times have different sizes or volumes or weights so this number is most of times meaningless.